### DISCUSSION TITLE
Is there documentation for the "package.contributes" field?

### DISCUSSION THREAD

**Discussion by zWingz** [eclipse-theia/theia#9581] [Q&A] [answered]
like
![image](https://user-images.githubusercontent.com/13031838/121675975-adacbd80-cae6-11eb-9519-8095aaa48c98.png)

**Comment by azatsarynnyy** [comment, marked as answer]
Hello @zWingz,
you can find it here https://code.visualstudio.com/api/references/contribution-points
